platforms/hyperledger-indy/configuration/roles/create/helm_component/node/templates/node.tpl

```diff
@@ -29,17 +29,17 @@ spec:
         repository: {{ network.docker.url }}/indy-node:0.3.0.0
     node:
       name: {{ component_name }}
-      INDY_NODE_IP: {{ stewardItem.node.port }}
-      INDY_CLIENT_IP: {{ stewardItem.client.port }}
-      ports:
-        indyNodePort: {{ stewardItem.node.port }}
-        indyClientPort: {{ stewardItem.client.port }}
+      ip: {{ stewardItem.publicIp }}
```

Comment:
should be 127.0.0.1 or 0.0.0.0, only the ambassador provisioned NLB will use that public IP, same for client.ip